tests/basic/storage_pool_test.go

```diff
@@ -1747,6 +1836,73 @@ var _ = Describe("{ResizeWithPXRestart}", func() {
 
 })
 
+var _ = Describe("{AddWithPXRestart}", func() {
+	//1) Deploy px with cloud drive.
+	//2) Create a volume on that pool and write some data on the volume.
+	//3) Expand pool by add-disk
+	//4) Restart px service where the pool expansion is in-progress
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("AddWithPXRestart", "Initiate pool expansion using resize-disk and restart PX while it is in progress", nil, 0)
+
+	})
+	var contexts []*scheduler.Context
+
+	stepLog := "should get the existing storage node and expand the pool by resize-disk"
+
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("adddskwrst-%d", i))...)
+		}
+		ValidateApplications(contexts)
+
+		stNodes := node.GetStorageNodes()
+		if len(stNodes) == 0 {
+			dash.VerifyFatal(len(stNodes) > 0, true, "Storage nodes found?")
+		}
+		stNode, err := GetRandomNodeWithPoolIOs(stNodes)
+		log.FailOnError(err, "error identifying node to run test")
+		selectedPool, err := GetPoolWithIOsInGivenNode(stNode)
+		log.FailOnError(err, "error identifying pool to run test")
+
+		stepLog := "Initiate pool expansion drive and restart PX"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+
+			poolToBeResized, err := GetStoragePoolByUUID(selectedPool.Uuid)
+			log.FailOnError(err, "Failed to get pool using UUID ")
```

Comment:
Maybe add the UUID that is used to get the StoragePool, would be good for debugging purposes
`log.FailOnError(err, "Failed to get pool using UUID [%s]", selectedPool.Uuid)`

Reply:
fixed